utils/custom.js

```diff
@@ -144,14 +147,27 @@ const handleAnonymousData = async (data, req, postAuthorId, myAnonymousId, anony
   };
 };
 
+const removePrefixTopic = (topicWithPrefix) => {
+  if (topicWithPrefix === null || topicWithPrefix === undefined) return;
+
+  if (topicWithPrefix.indexOf(NOTIFICATION_TOPIC_NAME_PREFIX) > -1) {
+    const topic = convertString(topicWithPrefix, NOTIFICATION_TOPIC_NAME_PREFIX, '');
+    return topic;
+  }
+
+  return topicWithPrefix;
+};
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

In the `removePrefixTopic` function, there's a potential issue where the function might return `undefined`. If `topicWithPrefix` is null or undefined, the function returns immediately without any value. It would be better to return an empty string in this case to ensure the function always returns a string.
```diff
- if (topicWithPrefix === null || topicWithPrefix === undefined) return;
+ if (topicWithPrefix === null || topicWithPrefix === undefined) return '';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->